package-lock.json

```diff
@@ -1,18722 +1,8 @@
 {
   "name": "generator-generator",
   "version": "5.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
we def don't want to rollback to lockfile v1, ideally we should move to lockfile v3 to avoid all the duplication 😅

Reply:
Yes! great suggestion. Implemented in 422579b11b8dff5c4fdc65dfdeda1d80ab030edc